profiles/common/modules/features/nexteuropa_trackedchanges/nexteuropa_trackedchanges.admin.inc

```diff
@@ -0,0 +1,700 @@
+<?php
+/**
+ * @file
+ * Hooks for nexteuropa_trackedchanges admin interfaces.
+ */
+
+/**
+ * Multisite WYSIWYG settings form.
+ */
+function nexteuropa_trackedchanges_settings_form($form, &$form_state) {
+  $profiles = _nexteuropa_trackedchanges_get_profiles();
+  $info_table = _nexteuropa_trackedchanges_render_info_table($profiles['info']);
+
+  $form['wysiwyg_profiles_info'] = array(
+    '#title' => t('WYSIWYG profiles: change tracking status'),
+    '#type' => 'item',
+    '#markup' => $info_table,
+    '#weight' => -10,
+    '#description' => t('<strong>Warning</strong>: The disabling of the "change
+      tracking" buttons in a profile will be blocked if any tracked changes 
+      exist in any fields using this profile.'),
+  );
+
+  $form['nexteuropa_trackedchanges_disable_track_on_create'] = array(
+    '#title' => t('Disable on create content pages.'),
+    '#type' => 'checkbox',
+    '#description' => t('Disable change tracking on create content pages.'),
+    '#default_value' => variable_get('nexteuropa_trackedchanges_disable_track_on_create'),
```

Comment:
The default value is in the feature as It has impact on the stability of the functionality